### PR TITLE
Mostly Stun duration fixes

### PR DESCRIPTION
**Pull request by l34Um1**

### CODE REVIEW

**Comment by Chronophylos** [comment]
Resolved the confict

**Comment by KarmaticNeutral** [comment]
Juggernaut already falls off in the late game, rather than nerfing his number of jumps to prevent him from being in the air so long would it be possible to change the bounce time trait? It's current;y 0.4 seconds per bounce which is why 40 bounces was so insane. What about something like 0.1 seconds and leave bounces with scepter at 40 so he's in the air for 4 seconds and still get the damage? Either that or buff the damage because past t2 bosses the jug ult is no longer a threat.

**Comment by Haganeko** [comment]
Most of the damage from jugg ult isn't from the slashes, it's from attacks while he's slashing. Reducing the time would need us to buff his BAT while ulting, to preserve his total damage.

**Comment by KarmaticNeutral** [comment]
True, what if we met in the middle? Maintain the new time (18 jumps at 0.4 sec a jump = 7.2s) and raised that to 36 jumps at 0.2 sec. This nerf cost Jug 4.6k dmg during the duration of his ult as is. ( 40 jumps at avg 212 dmg =~ 8400 || 18 jumps at the same 212 =~ 3800) So even if a larger portion of the damage is from attacking during ult, this is still too significant.

**Comment by l34Um1** [comment]
I would be very surprised if Juggernaut ends up too weak because of this change. The hero is inherently very powerful. I'll consider a buff if it's needed but I don't think he needs it atm

File: game/scripts/npc/abilities/juggernaut_omni_slash.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@
 			"02"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"omni_slash_jumps"			"3 6 9 14 32"
+				"omni_slash_jumps"			"3 6 9 12 18"
```
Comment (warpdragon) [inline game/scripts/npc/abilities/juggernaut_omni_slash.txt]: Honestly I don't think its' gonna hurt him too much. Omnislash is more about the initiation, the attack damage, and the invulnerability window. Having it last longer than it needs to is just overkill.

File: game/scripts/npc/abilities/magnataur_reverse_polarity.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@
 			"02"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"polarity_damage"		"50 125 200 350 1000"
+				"polarity_damage"		"50 125 200 300 400"
```
Comment (warpdragon) [inline game/scripts/npc/abilities/magnataur_reverse_polarity.txt]: Harsh.

File: game/scripts/npc/abilities/abyssal_underlord_atrophy_aura.txt [hunk 3]
```diff
@@ -31,7 +31,7 @@
 			"02"
 			{
 				"var_type"						"FIELD_INTEGER"
-				"damage_reduction_pct"			"4 16 28 40 52 70"
+				"damage_reduction_pct"			"4 16 28 40 40 40"
```
Comment (warpdragon) [inline game/scripts/npc/abilities/abyssal_underlord_atrophy_aura.txt]: Absolutely.

File: game/scripts/npc/abilities/magnataur_reverse_polarity.txt (hunk 2 above)
Comment (l34Um1) [inline game/scripts/npc/abilities/magnataur_reverse_polarity.txt]: RP would be insane while dealing 0 damage, it doesn't need 1000.

**Comment by Haganeko** [comment]
About Underlord - the aura needed a nerf,  but remember it does fall off pretty hard when people start to get more and more bonus damage.
On reverse polarity - could it go 275/425? At least level 5 is not that underwhelming this way.

**Pull request merged**